app/models/eligibilities/v3/evidence_utils.rb

```diff
@@ -7,6 +7,93 @@ module V3
     module EvidenceUtils
       extend ActiveSupport::Concern
 
+      # All possible states for evidence verification
+      # @return [Array<Symbol>] List of all possible states for the state machine
+      STATES = [
+        :initial,
+        :attested,
+        :pending,
+        :review,
+        :outstanding,
+        :verified,
+        :unverified,
+        :negative_response_received,
+        :determined,
+        :expired,
+        :denied,
+        :errored,
+        :closed,
+        :corrected,
+        :rejected
+      ].freeze
+
+      # Definition of all allowed state transitions
+      # @return [Hash] Map of event names to transition rules
+      # @example
+      #   STATE_TRANSITIONS[:attest][:from] # Returns array of states from which :attest is allowed
+      #   STATE_TRANSITIONS[:attest][:to]   # Returns the destination state after :attest event
+      STATE_TRANSITIONS = {
+        attest: {
+          from: [:pending, :attested, :review, :outstanding, :rejected, :unverified, :negative_response_received, :verified],
```

Comment:
the from arrays are the same for different transitions, but they are all in different orders. can we please standardize the order just for consistency